assets/js/type-defs/customer.ts

```diff
@@ -0,0 +1,52 @@
+export interface Address {
+	// eslint-disable-next-line camelcase
+	first_name: string;
+	// eslint-disable-next-line camelcase
+	last_name: string;
+	company: string;
+	// eslint-disable-next-line camelcase
+	address_1: string;
+	// eslint-disable-next-line camelcase
+	address_2: string;
+	city: string;
+	state: string;
+	postcode: string;
+	country: string;
+}
+
+export type ShippingAddress = Address;
+export type BillingAddress = Address;
+
+export interface AddressFieldConfiguration {
```

Comment:
This type exists in js/settings/shared/default-address-fields.js - should we import from there instead? Same for the other types in this file.

Reply:
Cool, I added this file as part of another (WIP) PR and copied it over, looks like your changes made it through before mine did. I'm happy to remove this file, however I like the type aliases for `BillingAddress` and `ShippingAddress` and `KeyedAddressFieldConfiguration` I added. I'll rename and move them over.

Reply:
Actually, on second thought, we need to differentiate between the `Address` with the entered values in, and the `Address` with configuration data in. This is why I had both `Address` and `AddressFields`. I will add the type back that is the address with string values entered by the user.